src/components/i18n/Composition.js

```diff
@@ -64,6 +65,15 @@ class Composition {
         return this.ma.getMinimalString();
     }
 
+    /**
+     * @private
+     */
+    nextKey() {
+        const ret = `key${this.keyIndex}`;
```

Comment:
We usually try to avoid abbreviations, where possible, as they can hinder readability.

Reply:
You would prefer something like "returnValue" perhaps?

Reply:
There's always good ol' `result`.

Reply:
Changed to "result"